packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -105,9 +105,18 @@ export const getAnonymousIntrinsics = () => {
 
   // 25.3.2.2 AsyncGeneratorFunction.prototype
   const AsyncGenerator = AsyncGeneratorFunction.prototype;
+
+  const asyncIterable = {
+    // eslint-disable-next-line no-restricted-globals, no-empty-function
+    async *[Symbol.asyncIterator]() {},
+  };
+
+  // eslint-disable-next-line @endo/no-polymorphic-call, no-restricted-globals
+  const asyncIterator = asyncIterable[Symbol.asyncIterator]();
+
   // 25.5.1 Properties of the AsyncGenerator Prototype Object
-  const AsyncGeneratorPrototype = AsyncGenerator.prototype;
-  const AsyncIteratorPrototype = getPrototypeOf(AsyncGeneratorPrototype);
+  const AsyncGeneratorPrototype = AsyncGenerator.prototype; // undefined on Hermes
+  const AsyncIteratorPrototype = getPrototypeOf(asyncIterator);
```

Comment:
Wouldn't it be sufficient to do 
```suggestion
  const AsyncIteratorPrototype = getPrototypeOf(AsyncGeneratorFunctionInstance());
```

Reply:
That's likely. Worth testing if it falls into the category of what Hermes supports, but looks equivalent to what we've tried here.

Reply:
the suggestion works too ^ but still causes the test to fail, i've looked into it here https://github.com/endojs/endo/pull/2220#issuecomment-2061839642 if any thoughts